README.md

```diff
@@ -29,6 +29,18 @@ module.exports = function (items) {
 }
 ```
 
+Or create an element without using template literals:
+
+```js
+// list.js
+var bel = require('bel/create-element')
+
+module.exports = function (items) {
+  return bel('ul', items.map(item => bel('li', item)))
```

Comment:
re: the arrow function here, @shama do you think the examples in the readme should stick to ES5 + template literals?